R/load_runner.R

```diff
@@ -30,8 +30,6 @@ run_load <- function(con, load, transformed_data, test_queries, path,
   verify_first_run(con, log_table, log_data)
   before <- test_queries(con)
   DBI::dbBegin(con)
-  transaction_active <- TRUE
-  on.exit(if (transaction_active) {DBI::dbRollback(con)})
```

Comment:
I also wanted to highlight this change as I removed it as it wasn't being hit by the code coverage.

So I had added this a bit ago to handle the case what happens if there is some unexpected error in the code causing it to stop before the transaction is committed or rolledback. Looking at the documentation for DBI it looks like this is not necessary.

> Disconnection from a connection with an open transaction effectively rolls back the transaction. All data written in such a transaction must be removed after the transaction is rolled back.